bitnami/fluentd/README.md

```diff
@@ -217,11 +217,15 @@ The following tables lists the configurable parameters of the fluentd chart and
-| `tls.key`                                 | Server Key                                                                                                     | Server private key content                                                                              |
-| `tls.existingSecret`                            | Existing secret with certificate content                                                                       | `nil`                                                                                                   |
+| `tls.enabled`                                   | Enable TLS/SSL encrytion for internal communications                                                           | `false`                                                                                                 |
+| `tls.autoGenerated`                             | Generate automatically self-signed TLS certificates.                                                           | `false`                                                                                                 |
+| `tls.aggregator.existingSecret`                 | Name of the existing secret containing the TLS certificates for the Fluentd aggregator                         | `nil`                                                                                                   |
+| `tls.forwarder.existingSecret`                  | Name of the existing secret containing the TLS certificates for the Fluentd forwarder                          | `nil`                                                                                                   |
+| `mountedTLS.enabled`                            | Enable the addition of TLS certificates                                                                        | `false`                                                                                                 |
+| `mountedTLS.caCertificate`                      | Ca certificate                                                                                                 | Certificate Authority (CA) bundle content                                                               |
+| `mountedTLS.certificate`                        | Server certificate                                                                                             | Server certificate content                                                                              |
+| `mountedTLS.key`                                | Server Key                                                                                                     | Server private key content                                                                              |
+| `mountedTLS.existingSecret`                     | Existing secret with certificate content                                                                       | `nil`                                                                                                   |
```

Comment:
Shouldn't these `mountedTLS.*` parameters be part of the `tls.*` object?

Reply:
Let me explain the problem and why I did this change.

This chart previously provided the values:
```
tls.enabled       
tls.caCertificate 
tls.certificate   
tls.key           
tls.existingSecret
```
But setting them didn't make any configuration change, they simply mounted the Certificates secret in `/certs` folder

As this feature simply "mounts" the certificates, I decided to rename them to mountedTLS, in case users are using that feature, for example, to provide fluentd with output certificates.

The `tls.*` settings are now exclusive to provide certs to encrypt the forwarder->aggregator communications while certificates with other purposes could be mounted using `mountedTLS.*`.

Reply:
With this context, I don't see any reason to maintain the `mountedTLS.*` parameter. Users can still user `extraVolumes` and `extraVolmeMounts` to add custom certificates for other purposes, but I don't see why we should maintain two different ways to add certificates.

Reply:
I agree, I have removed `mountedTLS.*` and updated the Changelog.